test/extended/cluster/utils.go

```diff
@@ -550,10 +554,93 @@ func CreateTemplates(oc *exutil.CLI, c kclientset.Interface, nsName, config stri
 	return nil
 }
 
+// CreateSimpleTemplates creates templates in user defined namespaces without tuningsets
+func CreateSimpleTemplates(oc *exutil.CLI, nsName, config string, template ClusterLoaderObjectType) error {
+	clusterAdminClientConfig := oc.AdminConfig()
+	restmapper := oc.AsAdmin().RESTMapper()
+
+	templateFile, err := mkPath(template.File, config)
+	if err != nil {
+		return err
+	}
+	e2e.Logf("We're loading file %v: ", templateFile)
+
+	data, err := ioutil.ReadFile(templateFile)
+	if err != nil {
+		return err
+	}
+
+	//if template.Parameters == nil {
+	//	e2e.Logf("Template environment variables will not be modified.")
+	//} else {
+	//	params := convertVariablesToString(template.Parameters)
+	//	allArgs = append(allArgs, params...)
+	//}
+
+	templateObj, err := runtime.Decode(unstructured.UnstructuredJSONScheme, data)
+	if err != nil {
+		return err
+	}
+	e2e.Logf("Decoded template: %v\n", templateObj)
+
+	dynamicClient, err := dynamic.NewForConfig(clusterAdminClientConfig)
+	if err != nil {
+		return err
+	}
+
+	temp := templateObj.(*unstructured.Unstructured)
+	parameters, ok := temp.Object["parameters"]
+	e2e.Logf("Parameters (%t): %v", ok, parameters)
+	params := parameters.([]interface{})
+	for i, p := range params {
+		e2e.Logf("Parameter index: \"%v\", value: \"%v\"", i, p)
+		field := p.(map[string]interface{})
+		e2e.Logf("Name: %v, Value: %v", field["name"], field["value"])
+		if field["name"] == "IDENTIFIER" {
+			field["value"] = "2"
+			var f interface{} = field
```

Comment:
since these values are pointers, this replacement is not needed

Reply:
Thanks!